### DISCUSSION TITLE
/bin/bash not working - cannot access terminal mode (used to work fine)

### DISCUSSION THREAD

**Discussion by ppriorfl** [qdm12/gluetun#2338] [Q&A] [answered]
Running latest image under portainer.

I am having issues with unhealthy and dropping VPN so while I'm tracking down that problem I need to run some terminal commands to reset some private tracker info.

I used to be able to use the portainer option to open terminal (/bin/bash) but now it doesn't work.

Is bash deprecated or removed in latest images?  Am I missing something?  I need to be able to run a curl command to set a cookie and currently cannot do so.

**Comment by frepke** [comment]
/bin/ash does the job

**Reply by ppriorfl** [reply]
I tried that but I get:

ash: curl: not found

**Reply by frepke** [reply, marked as answer]
Open the Gluetun terminal and install curl "apk add curl" I think.

**Reply by ppriorfl** [reply]
Thank you that worked.  Not sure why bash was removed - my linux knowledge is (obviously) very limited.  Will continue to troubleshoot my disconnects
